.github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md

```diff
@@ -3,10 +3,13 @@ name: Enviar software para revisión
 about: Usa esta plantilla para enviar un software a revisar en Español (Experimental)
 ---
 
+<!---
+A continuación, introduzca los valores de (1) el usuario de GitHub de la persona que envía el paquete (sustituyendo "@github_handle") y (2) la URL del repositorio (sustituyendo "https://repourl"). También se pueden especificar valores para autores de paquetes adicionales, sustituyendo "@github_handle1", "@github_handle2" - elimine esto si no lo necesita. NO BORRAR LOS SÍMBOLOS HTML. Austituir sólo "@github_handle" y "https://repourl". Este comentario puede ser eliminado una vez que haya sido leído y comprendido.
```

Comment:
```suggestion
A continuación, introduzca los valores de (1) el usuario de GitHub de la persona que envía el paquete (sustituyendo "@github_handle") y (2) la URL del repositorio (sustituyendo "https://repourl"). También se pueden especificar valores para autores de paquetes adicionales, sustituyendo "@github_handle1", "@github_handle2" - elimine esto si no lo necesita. NO BORRAR LOS SÍMBOLOS HTML (e.g., "<!--author1-->", "<!--end-author1-->"). Austituir sólo "@github_handle" y "https://repourl". Este comentario puede ser eliminado una vez que haya sido leído y comprendido.
```
Is "e.g.," correct here? I don't know if Latin abbreviations are expected in Spanish.